issue_02/script.js

```diff
@@ -0,0 +1,10 @@
+const ul = document.getElementById('js-ul');
+const a = document.createElement('a');
+a.href = '1.html';
+a.textContent = 'これです';
+const img = document.createElement('img');
+img.src = 'bookmark.png';
+img.alt = 'ブックマーク';
+a.insertAdjacentElement('afterbegin', img);
+const li = document.createElement('li');
+ul.appendChild(li).appendChild(a);
```

Comment:
IMO
This is completely my preference, so I'll leave it to you to decide if you want to fix it or not.
I felt it would be easier to read the code if the constants and the processing related to them were written separately.
```suggestion
const ul = document.getElementById('js-ul');
const li = document.createElement('li');
const a = document.createElement('a');
const img = document.createElement('img');

img.src = 'bookmark.png';
img.alt = 'ブックマーク';
a.href = '1.html';
a.textContent = 'これです';
a.insertAdjacentElement('afterbegin', img);
ul.appendChild(li).appendChild(a);

```

Reply:
**Thanks for the comment and approve!**
As you pointed out, I think it is easier to read by separating the variables from the processing for them.
I will try to keep this in mind in the future.